Artificial intelligence (AI). Date created/updated: December, 9, 2024.
[MATHLAB, R, Python: numpy, sklearn.model, pylab, sklearn.decomposition]. IF=4.1.
<img width="129" alt="GSU" src="https://github.com/spawar2/HIV_Machine_Learning_Techniques/assets/25118302/7b7ef408-a134-44d7-ba36-bfdac1c98baf">
<img width="130" alt="BMC" src="https://github.com/spawar2/HIV_Machine_Learning_Techniques/assets/25118302/9e4380d9-3988-4f3e-a115-005f8ca50914">

# HIV_Machine_Learning_Techniques collaboration with Dr. Robert. Harrison and Dr. Irene. Weber, Georgia State University, Atlanta, USA. https://csds.gsu.edu/profile/robert-harrison/ https://cas.gsu.edu/profile/irene-weber/ Mathematics Genealogy Project: https://www.mathgenealogy.org/id.php?id=294685 (The research was supported in part by the National Institutes of Health (NIH) grant GM062920 (ITW & RWH)). ^^^Georgia State University, College of Arts and Sciences, Department of Computer Science and Biology, Atlanta, United States of America (USA). Digital Object Identifier (DOI)=Online International Standard Book Number (ISBN).
BioMed Central BMC Bioinformatics. [Paper Link:](https://www.ncbi.nlm.nih.gov/pubmed/30343664)
[Poster Link:](https://www.claflin-computation.com/lab-journey?pgid=ktmii98q-daa60d9f-46a5-42e0-8d93-c31af6bd37f5)
Analysis of drug resistance in HIV protease, Shrikant Pawar, Chris Freas, Robert W. Harrison, and Irene T. Weber, Publication: BioMed Central BMC: Bioinformatics (Publication date: October 4), collaboration with Dr. Harrison and Dr. Weber, Georgia State University, Atlanta, USA. (The research was supported in part by the National Institutes of Health (NIH) grant GM062920 (ITW & RWH)). Dissertation, IF=3.3. Github, Article link, [Cited times: 18]^^^ DOI: 10.1186/s12859-018-2331-y, Volume: 19, Issue: 11, Pages: 262.
[Poster Link:](https://www.claflin-computation.com/lab-journey?pgid=ktmii98q-4d6981f1-b60c-4a57-aabd-4081ef87dd8d)
Dissertation published at: https://scholarworks.gsu.edu/items/c859be20-bf1b-470c-884a-118056c4d62a
[Scientific Computing Conference, Atlanta, Georgia, 1 October, 2017, United States of America (USA).. Analysis of Drug Resistance in HIV PR, by Pawar SD, Irene Weber, Christopher F, and Robert Harrison.](https://technology.gsu.edu/scientific-computing-day/conference-archive/#2017)
††
†Corresponding author. ††First author. †††Second author. ††††Third author. †††††author.
[Pawar, Shrikant, "Bioinformatics Techniques for Studying Drug Resistance In HIV and Staphylococcus Aureus." Dissertation, Georgia State University, 2019.
doi: https://doi.org/10.57709/14306337](https://scholarworks.gsu.edu/biology_diss/216/)

<img width="824" alt="HIV" src="https://github.com/spawar2/HIV_Machine_Learning_Techniques/assets/25118302/212d6e3f-b050-462b-b3d0-469f382ecb41">

Plot_Clustering (Autosaved).Rd: HIV-1 protease deep Ribonucleic acid sequencing data read, K Means, Hierarchial, AGNES: Agglomerative Nesting TRY DIVISIVE Clustering, Selecting optimum number of clusters using elbow method, clusGap, clustering vectors to export.
SVM.py: Support vector machine (SVM), principal component analysis (PCA).
selected function(genfromtxt, train_test_split, svm.SVC, PCA, svmClassifier_2d.predict, np.meshgrid, Z.reshape, knn, kmeans, clusGap, Mclust, NbClust, dist, cutree, agnes, sapply).

Testing: Inhibitor,	Accuracy,	Postive Prediction Value (PPV),	Recall,	F, Indinavir (IDV):	
0.979,	0.974,	0.985,	0.979.'


https://www.java.com/
https://www.php.net/
https://www.mysql.com/
https://www.mathworks.com/products/matlab.html

https://www.r-project.org/
https://www.bioconductor.org/packages/3.21/BiocViews.html#___Software
https://bioconductor.org/packages/release/bioc/html/GenomicFeatures.html
https://bioconductor.org/packages/release/bioc/html/clusterProfiler.html
https://www.bioconductor.org/packages/release/bioc/html/enrichplot.html
https://ggplot2.tidyverse.org/
https://cran.r-project.org/package=randomForest
https://topepo.github.io/caret/
https://bioconductor.org/packages/release/bioc/html/DESeq2.html
https://cran.r-project.org/package=missForest
https://bioconductor.org/packages/release/bioc/html/affy.html
https://bioconductor.org/packages/release/bioc/html/limma.html
https://rstudio.github.io/DT/
https://bioconductor.org/packages/release/bioc/html/edgeR.html
https://apps.cytoscape.org/apps/cytohubba
https://cytoscape.org/
https://string-db.org/
https://www.gsea-msigdb.org/
https://bioconductor.org/packages//2.13/bioc/html/BGmix.html
https://ggplot2.tidyverse.org/
https://www.htmlwidgets.org/
https://cran.r-project.org/package=cluster
https://cran.r-project.org/package=fpc
https://cran.r-project.org/package=survival
https://www.bioconductor.org/packages/release/bioc/html/multiClust.html
https://www.bioconductor.org/packages/release/bioc/html/GEOquery.html
https://cran.r-project.org/package=dendextend
https://cran.r-project.org/package=amap
https://www.bioconductor.org/packages/release/bioc/html/preprocessCore.html
https://www.bioconductor.org/packages/3.21/BiocViews.html#___Software
https://shiny.posit.co/
https://cran.r-project.org/package=ggbeeswarm
https://cran.r-project.org/package=speedglm
https://magrittr.tidyverse.org/

https://www.python.org/
https://scikit-learn.org/
https://pandas.pydata.org/
https://matplotlib.org/
https://www.tidyverse.org/
https://numpy.org/
https://www.tensorflow.org/
https://kivy.org/
https://pydicom.github.io/
https://docs.python.org/3/library/shlex.html
https://scipy.org/
https://scikit-image.org/
https://seaborn.pydata.org/
https://librosa.org/doc/
https://docs.python.org/3/library/glob.html
https://pytorch.org/
https://pydicom.github.io/
https://biopython.org/docs/dev/api/Bio.Blast.html
https://pypi.org/project/openai/0.26.5/
https://docs.python.org/3/library/tkinter.html
https://docs.fast.ai/
https://pypi.org/project/torchvision/
